source/whereAny.js

```diff
@@ -0,0 +1,52 @@
+import _curry2 from './internal/_curry2';
+import _has from './internal/_has';
+import _isEmpty from './isEmpty';
+
+
+/**
+ * Takes a spec object and a test object; returns true if the test satisfies
+ * the spec. Each of the spec's own properties must be a predicate function.
+ * Each predicate is applied to the value of the corresponding property of the
+ * test object. `whereAny` returns true if at least one of the predicates return true, false
+ * otherwise.
+ *
+ * `whereAny` is well suited to declaratively expressing constraints for other
+ * functions such as [`filter`](#filter) and [`find`](#find).
+ *
+ * @func
+ * @memberOf R
+ * @since v0.27.0
+ * @category Object
+ * @sig {String: (* -> Boolean)} -> {String: *} -> Boolean
+ * @param {Object} spec
+ * @param {Object} testObj
+ * @return {Boolean}
+ * @see R.propSatisfies, R.where
+ * @example
+ *
+ *      // pred :: Object -> Boolean
+ *      const pred = R.whereAny({
+ *        a: R.equals('foo'),
+ *        b: R.complement(R.equals('xxx')),
+ *        x: R.gt(R.__, 10),
+ *        y: R.lt(R.__, 20)
+ *      });
+ *
+ *      pred({a: 'foo', b: 'xxx', x: 8, y: 34}); //=> true
+ *      pred({a: 'xxx', b: 'xxx', x: 9, y: 21}); //=> false
+ *      pred({a: 'bar', b: 'xxx', x: 10, y: 20}); //=> false
+ *      pred({a: 'foo', b: 'bar', x: 10, y: 20}); //=> true
+ *      pred({a: 'foo', b: 'xxx', x: 11, y: 20}); //=> true
+ */
+var whereAny = _curry2(function where(spec, testObj) {
```

Comment:
`whereAny` probably needs a better naming

Reply:
I'm not a big fan of the name either, but I haven't come up with a good alternative.  Any suggestions?

Reply:
nope... 

`some` as the [ES `Array#some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) ?

Reply:
I'd prefer `whereAny` to just `some`, and `whereSome` doesn't do anything for me either.

Uggh!  Naming is always hard.

Reply:
- [ ] use `some`

Reply:
let's see if someone helps to clear this out :), other comments have been addressed.


- [ ] define naming

Reply:
@CrossEye any news here? 
Should we keep `whereAny`?

Reply:
I haven't heard any other good candidates.  Let's go with it.

Reply:
- [ ] use whereAny